### DISCUSSION TITLE
黑科技-中國版蝙蝠俠休旅車

### DISCUSSION THREAD

**Discussion by number32984** [1101-GS3073-C/Python-Community-GS3073-C#606] [General]
這台車能夠360度原地旋轉，可以比擬蝙蝠俠的戰車了吧，還是越野SUV，這台會成為沙漠中的王者吧，我很看好他的銷，說不定會熱銷道沙國!
https://news.cnyes.com/news/id/5054497

**Comment by magicmicheal** [comment]
老哥確診就好好休息，怎麼連發16篇貼文呢？然後要發就好好發嘛，內文可以多一點嗎，我都沒有點入網址的習慣，你這樣我怎麼掰留言，阿還有你看來白開心一場了，你看起來一堆眼睛。

**Comment by awehao** [comment]
比亞迪宣布 中國首款國產百萬級豪車報到！，這是新聞標題，給上面的同學參考，我覺得比起科技感的車子，我個人還是喜歡那種老派的車，只能說科技在走，品味要有
